stack_integration_test.go

```diff
@@ -440,9 +409,5 @@ func TestStackConverged(t *testing.T) {
 	require.ElementsMatch(t, deployments, stack.DeploymentNames)
 	require.NotNil(t, stack.LatestStackConfiguration)
 
-	for _, deployment := range deployments {
-		pollStackDeploymentStatus(t, ctx, client, stack.ID, deployment, "paused")
-	}
-
```

Comment:
Why was this removed? Is this only needed for testing beta code? Seems like we would still need the ability to poll stack deployment status for this test even after the refactor.